### PR TITLE
Added SgDatatablesBundle recipe

### PR DESCRIPTION
**Pull request by samuel4x4**
| Q             | A
| ------------- | ---
| License       | MIT

This PR adds recipes for the SgDatatablesBundle (PR https://github.com/stwe/DatatablesBundle/pull/716)

#SymfonyConHackday2017

### CODE REVIEW

File: sg/datatablesbundle/1.0/manifest.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+    "bundles": {
+        "Sg\\DatatablesBundle\\SgDatatablesBundle": ["all"]
+    },
+    "copy-from-recipe": {
+        "config/": "%CONFIG_DIR%/"
+    },
+    "composer-scripts": {
+        "make cache-warmup": "script",
+        "assets:install --symlink --relative %PUBLIC_DIR%": "symfony-cmd"
+    }
```
Comment (fabpot) [inline sg/datatablesbundle/1.0/manifest.json]: Looks like this is a copy/paste remaining from framework-bundle, which should be removed.

File: sg/datatablesbundle/1.0/config/routes/sg_datatables_bundle.yaml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+sg_datatables_bundle:
+    resource: "@SgDatatablesBundle/Controller/"
+    type:     annotation
```
Comment (fabpot) [inline sg/datatablesbundle/1.0/config/routes/sg_datatables_bundle.yaml]: I haven't had a look at the bundle, but don't we need a `prefix` here?

Comment (samuel4x4) [reply sg/datatablesbundle/1.0/config/routes/sg_datatables_bundle.yaml]: There wasn't a global bundle prefix, I just added one moving it from the controller annotation on https://github.com/stwe/DatatablesBundle/pull/716. I updated their documentation and now I'll proceed to update the flex recipe also. I think it's a good practice to have a global bundle routing prefix.

File: sg/datatablesbundle/1.0/manifest.json (hunk 1 above)
Comment (samuel4x4) [reply sg/datatablesbundle/1.0/manifest.json]: done

**Comment by samuel4x4** [comment]
Thank you @fabpot ;)

**Pull request merged**